src/locale/locales/es/messages.po

```diff
@@ -3755,64 +3755,64 @@ msgstr ""
 
 #: src/view/com/notifications/FeedItem.tsx:211
 msgid "liked your custom feed"
-msgstr ""
+msgstr "le gusta tu feed personalizado"
 
 #: src/view/com/notifications/FeedItem.tsx:178
 msgid "liked your post"
-msgstr ""
+msgstr "le gusta tu post"
 
 #: src/view/screens/Profile.tsx:222
 msgid "Likes"
 msgstr "Cantidad de «Me gusta»"
```

Comment:
```suggestion
msgstr "Me gusta"
```
I thought I'd included this one but apparently not... This is the suggested change in #5831 of changing the translation of the `Likes` tab to simply `Me gusta` so probably makes sense to include it here.